lib_shared_ui_res/src/main/res/values/colors.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <color name="purple_200">#FFBB86FC</color>
+    <color name="purple_500">#FF6200EE</color>
+    <color name="purple_700">#FF3700B3</color>
+    <color name="teal_200">#FF03DAC5</color>
+    <color name="teal_700">#FF018786</color>
+    <color name="black">#FF000000</color>
+    <color name="white">#FFFFFFFF</color>
+
+
+    <color name="divider_bg">#48636389</color>
+    <color name="primary_text">#212121</color>
+    <color name="secondary_text">#757575</color>
+    <color name="red_D7304D">#D7304D</color>
```

Comment:
why'd you choose to name the color resources this way

Reply:
I was trying to think of a more descriptive name for the colors but I couldn't come up with any. So I settled for that

Reply:
What happens if you remove the hex codes from the color name

Reply:
Also if you try to do something and you are not making headway, you should ask for help, instead of settling for a  solution that isn't very proper

Reply:
Okay, thank you very much. I'll ask questions when there's no headway. I wanted to remove the hex code attached to the color name, but I was considering a case where I might need to add another shade/variation of the same color

Reply:
If you need to add a new variation then you can follow the format used in naming the purple colors

Reply:
Okay. I'll do that, thank you very much

Reply:
Please a question; the numbers appended to the purple colors (200, 500, etc)... is there a way I determine those numbers? For example, when there another shade of red to add, how can I know if it's 200, 300 etc from the hex code? Or I can give it an arbitrary number that makes sense for me? I tried to check on it, not found anything so far

Reply:
You can give it a value based on whether it's a darker or lighter shade. Lesser numbers for lighter shades and higher ones for darker shade. You can also use multiples of 100 like in the purple case

Reply:
Okay cool. Thanks alott!